distrobox-create

```diff
@@ -166,8 +166,19 @@ generate_command() {
 
 # First, check if the image exists in the host
 if ! podman image exists "${container_image}"; then
-	echo "Image not found, run this command first:"
-	echo "	podman pull ${container_image}"
+        echo "Image not found."
+    read -n1 -p "Do you want to pull the image now?[y/n] " yn
```

Comment:
As above, check if you can use stderr for this (maybe splitting between an echo and a read) 

If the user says yes and the pull is successful, we should continue with the creation and we need a valid command for that